server/src/main/java/org/apache/cassandra/sidecar/acl/authorization/BasicPermissions.java

```diff
@@ -79,4 +79,5 @@ public class BasicPermissions
 
     // Live Migration permissions
     public static final Permission LIST_FILES = new DomainAwarePermission("LIVE_MIGRATION:LIST_FILES", CLUSTER_SCOPE);
+    public static final Permission STREAM_FILES = new DomainAwarePermission("LIVE_MIGRATION:STREAM", CLUSTER_SCOPE);
```

Comment:
I would keep a single permission for the feature , similar to what we have for CDC

Reply:
Created a separate permission so that STREAM_FILES can be given only to Sidecars and LIST_FILES can be given to users. Doesn't it make sense?

Reply:
Proposing to keep the stream permission to Sidecar only by default. From CEP-40, 

<img width="1249" alt="image" src="https://github.com/user-attachments/assets/d7409d44-1ace-439f-b7cc-1155e4deb2f2" />
list APIS can be exposed to admin/operator (for debugging purposes).  admin/operator can check the list, but may not be required to download the files.

Reply:
Make sense. Consider adding comment in the live migration permissions to explain the expected scenario.